### ISSUE TITLE
Improve documentation for I2C blocks

### ISSUE THREAD

**Issue by ppelleti** [Microsoft/pxt-microbit#1314]
**Is your feature request related to a problem? Please describe.**
This came up when someone asked a question on the micro:bit Slack.  He was trying to use the I2C read and write blocks to read and write an I2C EEPROM.  In helping him, I was a bit annoyed that the documentation on the read and write blocks was minimal.  In particular, I had to go to the [Runtime documentation](https://lancaster-university.github.io/microbit-docs/ubit/i2c/) to fully understand what the "repeated" parameter meant.

**Describe the solution you'd like**
1. It would be helpful to have a little more text to explain to the uninitiated that these are I2C bus reads and writes, and not memory reads and writes.
2. Document the "repeated" parameter.
3. Explain the difference between Int8LE and Int8BE.  (How does a single byte have an endianness?)

**Describe alternatives you've considered**
An alternative to documenting the "repeated" parameter would be to rename it "send stop bit" (or perhaps "end of transaction") and invert the sense of it.  I think that would be clearer to those familiar with I2C.  (And I'm not sure the I2C blocks will be all that useful in any case to those unfamiliar with I2C.)

If there's no difference between Int8LE and Int8BE, then just replace them with a single "Int8" choice.

**Additional context**
None.

**Comment by ppelleti** [comment]
Just a followup: I noticed that the repeated parameter *is* documented [here](https://makecode.microbit.org/reference/pins/i2c-read-number), so I just didn't RTFM enough.  The "documentation" I'm referring to in the report above is the mouse-over documentation for the block.  (i. e. the doc comment)

Also, I found the documentation for [NumberFormat](https://makecode.microbit.org/types/buffer/number-format), but that still didn't answer my question of why single-byte numbers are available in both big endian and little endian.
